libs/build_nceplibs.sh

```diff
@@ -247,8 +247,12 @@ fi
 
 cd ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
 
-# Download CRTM fix files
 if [[ "$name" == "crtm" ]]; then
+# Correction to allow for the crtm-2.4.0 serial build:
+  if [[ "${version}" == "v2.4.0" ]]; then
+sed -i'.orig' -e 's/  find_package(MPI REQUIRED)/\#  find_package(MPI REQUIRED)/g' ./cmake/FindNetCDF.cmake
+  fi
```

Comment:
Is the intent for v2.4.0 *never* be able to use MPI, or to make it optional?

Reply:
The current installation in hpc-stack is set for the serial installation only of the crtm.  Only the libraries nemsiongfs, nemsio, w3emc, wrf_io have options of `using_mpi `set to "YES" or "NO". This "using_mpi" variable further determines whether MPI compilers or serial compiler are use for building the library. In case of  `crtm` these are serial compilers.
   A new version of `crtm/2.4.0` checks the netcdf version. The netcdf module loaded is a parallel one if it's available. If parallel netcdf version is detected, parallel built of the `crtm` is also required - but the compilers passed by cmake are the serial ones. This throws the error during the crtm/2.4.0 build. 
   
   A recipe for building the `crtm` may likely be modified in build_nceplibs.sh to allow either parallel or serial installation, if needed.